dash/frontend/src/app/modules/private/pages/cluster/kube-bench/kube-bench-report-details/kube-bench-report-details.component.scss

```diff
@@ -1,47 +1,75 @@
-.test-main {
-  max-width: 8%;
-}
+.control-accordions {
 
-.test-description-main {
-  min-width: 62%;
-  max-width: 62%;
-}
+  > div > mat-expansion-panel {
```

Comment:
It would be better to use a class than direct child rules.

Reply:
Unfortunately, the class is overwritten by default styling rules if it's not targeted directly.